gitlab2rbac.py

```diff
@@ -38,11 +39,12 @@ def connect(self):
             exception: If any errors occurs.
         """
         try:
-            self.client = Gitlab(url=self.url, private_token=self.token,
-                                 timeout=self.timeout)
+            self.client = Gitlab(
+                url=self.url, private_token=self.token, timeout=self.timeout
+            )
```

Comment:
This diff is ugly tho, I prefer the old version much more. I wonder if there's any way of configuring this in black.